docs/plugins/plugin-development.md

```diff
@@ -40,7 +40,7 @@ import { createRouteRef } from '@backstage/core-plugin-api';
 
 // Note: This route ref is for internal use only, don't export it from the plugin
 export const rootRouteRef = createRouteRef({
-  title: 'Example Page',
+  id: 'Example Page',
```

Comment:
Just a note for posterity that we could perhaps have a nicer example here, but it doesn't actually matter much. In the new frontend system we're getting rid of route ref IDs altogether

Reply:
okay cool good to know!